dockerfiles/compile-extension/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.7'
+
+services:
+
+  7.3-alpine:
+    image: datadog/dd-trace-ci:php-compile-extension-alpine-7.3
+    build:
+      context: .
+      dockerfile: Dockerfile_alpine
+      args:
+        php_version: 7.3.2
```

Comment:
This is quite old; is there a specific reason for using it instead of the latest (7.3.23 atm)?

Reply:
yep it was there from back in the days, updating it, thanks.